docs/sources/mimir/operators-guide/deploy-grafana-mimir/jsonnet/configure-deployment-mode.md

```diff
@@ -0,0 +1,104 @@
+---
+description: Learn how to configure Grafana Mimir deployment mode.
+menuTitle: Configure deployment mode
+title: Configure deployment mode
+weight: 40
+---
+
+# Configure deployment mode
+
+Grafana Mimir supports multiple [deployment modes](). By default the provided jsonnet will deploy in microservices mode. Monolithic mode is not current supported in jsonnet.
+
+## Use Read-Write deployment mode
+
+> **Warning:**
+> Read-Write deployment mode is currently considered experimental and not ready for production use.
+
+### Requirements
+
+Read-Write deployment mode requires that multi-zone ingesters and multi-zone store gateways are used. Additionally rule evaluation is performed within Mimir-Backend, so ruler remote evaluation must be disabled.
```

Comment:
```suggestion
Read-write deployment mode requires that multi-zone ingesters and multi-zone store gateways are used. Additionally rule evaluation is performed within Mimir-Backend, so ruler remote evaluation must be disabled.
```

Reply:
> Additionally rule evaluation is performed via Mimir read, so ruler remote evaluation must be disabled.

This is not 💯 correct. We still do remote rule evaluation, via mimir **read**, but we don't run a dedicated read-path stack for rule evaluations.